### PR TITLE
CB-9421 Fixed searchpath parameter being ignored

### PR DESCRIPTION
**Pull request by alsorokin**
https://issues.apache.org/jira/browse/CB-9421

### CODE REVIEW

File: cordova-lib/src/plugman/fetch.js [hunk 1]
```diff
@@ -236,7 +236,7 @@ function findLocalPlugin(plugin_src, searchpath, pluginInfoProvider) {
 
     versions.forEach(function(pinfo) {
         // Ignore versions that don't satisfy the the requested version range.
-        if (!semver.satisfies(pinfo.version, versionspec)) {
+        if (!semver.satisfies(pinfo.version.replace(/-dev$/, ''), versionspec)) {
```
Comment (vladimir-kotikov) [inline cordova-lib/src/plugman/fetch.js]: This is not obvious, why this change is required. Could you please add a link to JIRA/semver docs or short explanation, why this needed.

**Comment by purplecabbage** [comment]
lgtm!
